docs/index.md

```diff
@@ -30,7 +30,7 @@ _**We only give support for the latest release/dev builds; please update before
 * #### Chat
     * [`Herochat`](https://www.spigotmc.org/resources/herochat.34305/updates), [`LegendChat`](https://www.spigotmc.org/resources/legendchat.6268/), [`LunaChat`](https://github.com/ucchyocean/LunaChat), [`TownyChat`](https://www.spigotmc.org/resources/towny-advanced.72694/), [`VentureChat`](https://www.spigotmc.org/resources/venturechat.771/)
 * #### Vanish
-    * [`Essentials`](https://www.spigotmc.org/resources/essentialsx.9089/), [`PhantomAdmin`](https://www.spigotmc.org/resources/phantomadmin.37845/), [`SuperVanish`](https://www.spigotmc.org/resources/supervanish-be-invisible.1331/), [`VanishNoPacket`](https://dev.bukkit.org/projects/vanish)
+    * [`Essentials`](https://www.spigotmc.org/resources/essentialsx.9089/), [`PhantomAdmin`](https://www.spigotmc.org/resources/phantomadmin.37845/), [`SuperVanish`](https://www.spigotmc.org/resources/supervanish-be-invisible.1331/), [`VanishNoPacket`](https://dev.bukkit.org/projects/vanish), [`VelocityVanish`](https://www.spigotmc.org/resources/velocityvanish-1-8-1-19-3-no-database-required.105992/)
```

Comment:
This section is for Vanish plugins that DiscordSRV has made hooks for, not for plugins that use our API. Please move your addition to the `### Plugins that use our API:` section.